src/main/java/de/maxhenkel/pipez/blocks/EnergyPipeBlock.java

```diff
@@ -22,6 +22,9 @@ protected EnergyPipeBlock() {
 
     @Override
     public boolean canConnectTo(LevelAccessor world, BlockPos pos, Direction facing) {
+        if (!super.canConnectTo(world, pos, facing)) {
```

Comment:
Adding `super.canConnectTo` is not very clean in my opinion. `canConnectTo` should only be for the implementation of specific pipe types and the config check should be done in another method thats in the base PipeBlock class.

(Same for `FluidPipeBlock`, `GasPipeBlock`, `ItemPipeBlock` and `UniversalPipeBlock`)